lib/orm/queries.rb

```diff
@@ -0,0 +1,48 @@
+module Matrack
+  class Queries < FetchQueries
+    class << self
+      def destroy(id)
+        execute "DELETE FROM #{table_name} WHERE (#{uniq_id} = #{id})"
+      end
+
+      def destroy_all
+        execute "DELETE FROM #{table_name}"
+      end
+
+      def count
+        row = db_conn.execute "SELECT COUNT(*) FROM #{table_name}"
+        row.hash_getter.values.first
+      end
+
+      # .update(1, title: "updated task again", done: "yes")
+      def update(id, hash)
+        fields = hash.map{|k,v| "'#{k}'" " = "  "'#{v}'"}.join(", ")
```

Comment:
Unnecessary spacing detected.<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.